sbin/pfdhcplistener

```diff
@@ -252,8 +305,17 @@ sub process_pkt {
                         $args{inline_sub_connection_type} = $NET_TYPE_INLINE_L3;
                    }
                 }
-                my $apiclient = pf::api::queue->new(queue => 'pfdhcplistener');
-                $apiclient->notify('process_dhcp', %args, udp_payload_b64 => $udp_payload_b64);
+                my $rate_limiting = $Config{network}->{dhcp_rate_limiting};
+                if($rate_limiting == 0 || !$rate_limit_cache->get($rate_limit_key)) {
+                    my $apiclient = pf::api::queue->new(queue => 'pfdhcplistener');
+                    $apiclient->notify('process_dhcp', %args, udp_payload_b64 => $udp_payload_b64);
+                    if($rate_limiting != 0) {
+                        $rate_limit_cache->set($rate_limit_key, 1, $rate_limiting);
+                    }
+                }
+                else {
+                    $logger->info("Ignoring packet due to rate-limiting");
```

Comment:
This could create a lot of noise in the log files can you change to debug or trace.